tests/e2e/scenarios/test_reborn_webui_v2_legacy_projects.py

```
@@ -0,0 +1,402 @@
+            "engineering, and sales."
+        ),
+        "metadata": {"goals": ["Ship v2.0 by June 15", "Hit launch signups"]},
+        "state": "active",
+        "role": "owner",
+        "created_at": "2026-04-11T08:45:00Z",
+        "updated_at": "2026-04-12T08:45:00Z",
+    },
+]
+
+
+async def _open_mocked_projects_page(reborn_v2_server, reborn_v2_browser):
```

Ruff (0.15.20)

[warning] Too many statements (56 > 50)

(PLR0915)